buildrunner/config/fetch/__init__.py

```diff
@@ -23,23 +24,26 @@ def fetch_file(url, config):
     # config) => bool)
 
     parsed_url = urllib.parse.urlparse(url)
+    scheme = parsed_url.scheme
```

Comment:
This is interesting to me! I didn't know this could come back as bytes, whats the case that happens in ?

Reply:
That's a really good question. I hit it during testing, and I still have no idea why it was that way. I think it might be a specific version of python that did it like 3.8 or something.